bucket/klatexformula.json

```diff
@@ -0,0 +1,30 @@
+{
+    "version": "4.0.0",
+    "homepage": "https://klatexformula.sourceforge.io",
+    "description": "An easy-to-use graphical application for generating images from LaTeX equations.",
+    "license": "GPL-2.0-or-later",
+    "depends": "latex",
```

Comment:
Where is mentioned you need to have miktex installed?

Reply:
https://klatexformula.sourceforge.io/downloads#system-requirements

Reply:
> On Windows, you **can** install

Why do you force user to have Miktex installed? Why I cannot use installation of Texlive?

Reply:
I'll change it to suggest.